packages/mui-material/src/Tabs/Tabs.js

```diff
@@ -820,44 +848,68 @@ const Tabs = React.forwardRef(function Tabs(inProps, ref) {
 
   const conditionalElements = getConditionalElements();
 
+  const [RootSlot, rootSlotProps] = useSlot('root', {
+    ref,
+    className: clsx(classes.root, className),
+    elementType: TabsRoot,
+    externalForwardedProps: {
+      ...externalForwardedProps,
+      ...other,
+      component,
+    },
+    ownerState,
+  });
+
+  const [ScrollerSlot, scrollerSlotProps] = useSlot('scroller', {
+    ref: tabsRef,
+    className: classes.scroller,
+    elementType: TabsScroller,
+    externalForwardedProps,
+    ownerState,
+    additionalProps: {
+      style: {
+        overflow: scrollerStyle.overflow,
+        [vertical ? `margin${isRtl ? 'Left' : 'Right'}` : 'marginBottom']: visibleScrollbar
+          ? undefined
+          : -scrollerStyle.scrollbarWidth,
+      },
+    },
+  });
+
+  const [FlexContainerSlot, flexContainerSlotProps] = useSlot('flexContainer', {
```

Comment:
The `flexContainer` slot seems difficult to understand from the users perspective, would 'list' be a better name?

Reply:
I don't think so, this is because the slot is created with `flexContainer` name. It's not a new slot.

```js
// line: 198
const FlexContainer = styled('div', {
  name: 'MuiTabs',
  slot: 'FlexContainer',
  overridesResolver: (props, styles) => {
    const { ownerState } = props;
    return [
      styles.flexContainer,
      ownerState.vertical && styles.flexContainerVertical,
      ownerState.centered && styles.centered,
    ];
  },
})({
```

So styleOverrides already work with `flexContainer` slot. I think `slots` and `slotProps` should follow the same name.

Reply:
Could we deprecate `styles.flexContainer` and change it? I really think it's a bad name as it doesn't explain to users what this slot's role is in the component: `flexContainer` can be anything.

Reply:
I agree it's not a good name. Let's change it now that we are at it 🙏